packages/trpc/server/routers/viewer/eventTypes/create.handler.ts

```diff
@@ -83,6 +83,22 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
       },
     };
     data.schedulingType = schedulingType;
+
+    if (!isManagedEventType) {
+      data.hosts = {
+        create: [
+          {
+            user: {
+              connect: {
+                id: userId,
+              },
+            },
+            isFixed: data.schedulingType === SchedulingType.COLLECTIVE,
+            priority: 2, //default
+          },
+        ],
+      };
```

Comment:
Add owner of the event type to host when creating event type

Reply:
not sure if we want that? What if we just don't show any availability when no one is assigned, that would make the most sense to me

Reply:
i added this check in util.ts to throw error and not display any slots.

Reply:
this is just to prevent a scenario when no host is assigned when a team event type is created and we already add the owner to users[] field in event type

Reply:
What I meant is, is that I would prefer to not have any host assigned after creating the event type

Reply:
@CarinaWolli yeah but a lot of people get confused when they create team event type and go to booking page and get error Could not book the meeting "Cannot read properties of undefined (reading 'name')

Reply:
and then they contact intercom

Reply:
I think what we should do is, is not having any user assigned when creating by changing [line 60](https://github.com/calcom/cal.com/pull/13888/files#diff-6b49c95fdbabc50de1c8646d7e179f252070e18047a135d010052e7665ea5c16R60): 



`users: isManagedEventType || schedulingType ? undefined : { connect: { id: userId } },`